src/gui/MainWindow.cpp

```diff
@@ -893,6 +893,10 @@ void MainWindow::updateRecentlyOpenedProjectsMenu()
 		{
 			m_recentlyOpenedProjectsMenu->addAction(
 					embed::getIconPixmap( "project_file" ), *it );
+#ifdef LMMS_BUILD_APPLE
+			m_recentlyOpenedProjectsMenu->actions().last()->setIconVisibleInMenu(false); // Workaround for https://bugreports.qt.io/browse/QTBUG-44565
```

Comment:
I feel this would be cleaner

```cpp
m_recentlyOpenedProjectsMenu->actions().last()->setIconVisibleInMenu(false); // Per QTBUG-44565
m_recentlyOpenedProjectsMenu->actions().last()->setIconVisibleInMenu(true);
```